### PR TITLE
115 update non UUID pattern

### LINKED ISSUE #115: Adding a string instead of real UUID's in giftaid-update url is going to a sorry page
Eg: https://giftaid-staging.comicrelief.com/update/test is going to a sorry page.

it accepts only real UUID's

https://dashboard.iopipe.com/project/65d0875a-e96a-4a1e-8a08-976790bf45d2/alert/1001a1af-ae07-445f-af4d-139d2db0f7d8/event/8644a375-dd0c-4541-8baf-18db465632ef

### PR DESCRIPTION
**Pull request by s-barrah**
Fix #115 
- Validates non UUID / invalid strings parameters in the update url

### CODE REVIEW

**Comment by adamclark-dev** [comment]
@s-barrah thought we removed the uuid checking on update, as we were getting errors when the update email went out as GAPS ID is not UUID.

**Comment by s-barrah** [comment]
> @s-barrah thought we removed the uuid checking on update, as we were getting errors when the update email went out as GAPS ID is not UUID.

On the backend it validates if not a valid UUID. This fix simply prevents it from getting submitted to the backend. Currently the user gets directed to the sorry page.

File: src/pages/UpdateForm/UpdateForm.js [hunk 1]
```diff
@@ -260,6 +264,11 @@ class UpdateForm extends Component {
 
     let item;
     let allClasses;
+    // Scroll to url trans Id error message
+    if (this.state.urlTransIDErrorMessage) {
+      document.querySelector('#field-error--urlTransID').scrollIntoView('smooth');
+      // window.scrollTo(0, this.myRef.current.offsetTop);
+    }
```
Comment (Heleen-cr) [inline src/pages/UpdateForm/UpdateForm.js]: commented out line

**Comment by KrupaPammi** [comment]
@s-barrah from QA POV, tests looks good to me. @adamclark-dev  If all good with the code for this issue, would be good to get this merged please, thanks.

**Pull request merged**